### PR TITLE
`Development`: Increase client test coverage of logs

### PR DESCRIPTION
**Pull request by coaxial1**
#### General
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).

### Motivation and Context

Increase coverage for ``app/admin/logs``.

### Test Coverage
| | Branch % | Lines % |
| - | - | - |
| Before | 33 | 77 |
| After | 76 | 100 |

#### Code Review
- [x] Review 1
- [x] Review 2

### CODE REVIEW

File: src/test/javascript/spec/component/admin/logs.component.spec.ts [hunk 1]
```diff
@@ -30,58 +28,111 @@ describe('Component Tests', () => {
...
-                            main: {
-                                effectiveLevel: 'WARN',
-                            },
+        it('should set all default values correctly', () => {
+            expect(comp.filter).toBe('');
+            expect(comp.orderProp).toBe('name');
+            expect(comp.ascending).toBe(true);
+        });
+
+        it('Should call load all on init', () => {
```
Comment (b-fein) [inline src/test/javascript/spec/component/admin/logs.component.spec.ts:37]:
```suggestion
        it('should call load all on init', () => {
```

**Review by b-fein — APPROVED** [review]

File: src/test/javascript/spec/component/admin/logs.component.spec.ts (hunk 1 above)
Comment (coaxial1) [reply src/test/javascript/spec/component/admin/logs.component.spec.ts]: Added in bfc19df :)

**Review by b-fein — APPROVED** [review]

**Review by akesfeden — APPROVED** [review]
Added tests look good.

File: src/test/javascript/spec/component/admin/logs.component.spec.ts [hunk 2]
```diff
@@ -30,58 +28,111 @@ describe('Component Tests', () => {
...
+                    },
+                } as unknown as LoggersResponse),
+            );
+
+            // WHEN
+            comp.changeLevel('main', 'ERROR');
+
+            // THEN
+            expect(service.changeLevel).toHaveBeenCalled();
+            expect(service.findAll).toHaveBeenCalled();
```
Comment (willergeorg) [inline src/test/javascript/spec/component/admin/logs.component.spec.ts:77]: Same here

**Review by willergeorg — APPROVED** [review]
Marked you some nits. Besides that, lgtm.

File: src/test/javascript/spec/component/admin/logs.component.spec.ts [hunk 3]
```diff
@@ -30,58 +28,111 @@ describe('Component Tests', () => {
...
-                comp.ngOnInit();
+            // WHEN
+            comp.ngOnInit();
 
-                // THEN
-                expect(service.findAll).toHaveBeenCalled();
-                expect(comp.loggers?.[0]).toEqual(expect.objectContaining(log));
-            });
+            // THEN
+            expect(service.findAll).toHaveBeenCalled();
```
[the remark this answers is not in the document]
Comment (coaxial1) [reply src/test/javascript/spec/component/admin/logs.component.spec.ts]: Added in 18cb9f7

File: src/test/javascript/spec/component/admin/logs.component.spec.ts (hunk 2 above)
Comment (coaxial1) [reply src/test/javascript/spec/component/admin/logs.component.spec.ts]: Added in 18cb9f7

**Review by b-fein — APPROVED** [review]

**Review by akesfeden — APPROVED** [review]
Re-approving after changes.

**Review by willergeorg — APPROVED** [review]

**Pull request merged**